src/main/java/duke/storage/TaskList.java

```diff
@@ -0,0 +1,123 @@
+package duke.storage;
+
+import duke.DukeExceptions;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+<<<<<<< HEAD
+=======
+/**
+ * List class to hold the listed Tasks.
+ */
+>>>>>>> branch-A-JavaDoc
+public class TaskList extends ArrayList<Task> {
+
+    private static final String DIVIDER = "%%";
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        result.append("Here are the tasks in your list! \n");
+        int index = 1;
+        for (Task task : this) {
+            result.append(index)
+                    .append(". ")
+                    .append(task.toString())
+                    .append('\n');
+            index++;
+        }
+        return (result.toString().stripTrailing());
+    }
+
+    /**
+     * Add all of the task in the List of String.
+     *
+     * @param inputString The list of lines of string.
+     */
+    public void importFromList(List<String> inputString) {
+        this.clear();
+        for (String line : inputString) {
+            String[] splitString = line.split(DIVIDER, 0);
+            Task task;
+            if (splitString[0].equals("T")) {
+                task = ToDo.create(splitString[2]);
+
```

Comment:
Is this newline necessary before "} else if (..) {"? Same for the rest of the if-else block